portal/src/graphql/portal/ApplicationsConfigurationScreen.module.scss

```diff
@@ -55,3 +55,19 @@
   pointer-events: none;
   opacity: 0.5;
 }
+
+.callout {
+  padding: 8px;
+}
+
+.copyButtonRoot {
+  margin: 4px;
+}
+
+.copyButtonRoot:hover {
+  background-color: #d8d6d3;
+}
+
+.copyButtonRoot:active {
+  background-color: #c2c0be;
+}
```

Comment:
Changing the hover and active color is of customizing the component. So this should be written using `styles` prop.